tiatoolbox/models/architecture/micronet.py

```diff
@@ -333,13 +363,16 @@ def out_arch_branch(in_ch, num_class=2):
 
 class MicroNet(ModelABC):
     """Initialise MicroNet [1].
-    The following models have been included in tiatoolbox.
-    1. `micronet_hovernet-consep`: This is trained on
-    `ConSep dataset <https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet/>`_
-    The model is retrained in torch as the original model with results on ConSep [2]
-    was trained in TensorFlow.
 
-    The tiatoolbox model should produce following results on the ConSep dataset.
+    The following models have been included in tiatoolbox:
+    1. `micronet_hovernet-consep`:
```

Comment:
@shaneahmed should be changing the naming here - I assume in another PR?

Reply:
@shaneahmed Any suggestions?